vendor-bin/doctrine_mongodb/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "autoload": {
```

Comment:
I'm not sure to understand what you mean.
However, i've just replicated the `composer.json` in `vendor-bin/doctrine` folder and adapted the requirements, i thought it was ok...